scripts/upload_asset.sh

```diff
@@ -0,0 +1,93 @@
+#!/bin/bash
```

Comment:
Maybe it should be in `ci` folder like it was before? `scripts` sounds like something users can use or distros think that they should package, however it's only useful and will be used by our CI job.

Reply:
Having a ci folder just for this one script seems a bit much. Why do you think that `scripts` is for user scripts?

Reply:
Well, how distribution will understand that this shouldn't be packaged? It seems like some generic script, well, they can look into it, but having `ci` will clarify things for folks that are looking at the repo first time.

Reply:
Why would a distribution start packaging random scripts?

Reply:
I just know that certain distros are packaging our scripts already, so they'll just package this one as well...

Reply:
They are? Do you know which?

Reply:
https://gitweb.gentoo.org/repo/gentoo.git/tree/x11-terms/alacritty/alacritty-0.5.0.ebuild#n360